CIP-param-application/README.md

```diff
@@ -0,0 +1,113 @@
+---
+CIP: ????
+Title: On-Chain Parameter Application
```

Comment:
It would be good if now and/or at the meeting we could settle on a CIP title the same as the PR title.  We've had to remove back-ticked expressions from CIP titles recently (#930, to fix portability problems) so the next best choice would either be this expression above or one that concisely describes the new builtin (instead of quoting its name).

Reply:
resolved with https://github.com/cardano-foundation/CIPs/pull/934#event-15240537419.